app/src/main/java/org/mozilla/fenix/wallpapers/LegacyWallpaperMigration.kt

```diff
@@ -106,9 +109,28 @@ class LegacyWallpaperMigration(
         return@withContext migratedWallpaperName
     }
 
+    /**
+     * Helper function used to migrate a legacy wallpaper's card colors that previously did not exist.
+     */
+    fun migrateExpiredWallpaperCardColors(wallpaperName: String) {
+        settings.shouldMigrateLegacyWallpaperCardColors = false
+
+        if (wallpaperName == TURNING_RED_MEI_WALLPAPER_NAME) {
+            settings.currentWallpaperCardColorLight = TURNING_RED_MEI_WALLPAPER_CARD_COLOR_LIGHT.toLong(radix = 16)
```

Comment:
Out of scope, but I'm wondering if an extension function to abstract this `toLong(radix = 16)` pattern might be useful. Something like `toHexColorLong` could be more informative IMO

Reply:
Sure, I'll file a follow-up

Reply:
https://github.com/mozilla-mobile/fenix/issues/27575